src/OrchardCore.Modules/OrchardCore.Email.Azure/Manifest.cs

```diff
@@ -1,14 +1,14 @@
 using OrchardCore.Modules.Manifest;
 
 [assembly: Module(
-    Name = "Azure Email Provider",
+    Name = "Azure Communication Services (Email)",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Provides an email service provider leveraging Azure Communication Services (ACS).",
+    Description = "Provides email service providers leveraging Azure Communication Services (ACS).",
```

Comment:
Too many "provide".

```suggestion
    Description = "Adds email service providers leveraging Azure Communication Services (ACS).",
```

Reply:
the term "Providers" is the common term we use when describing feature.

Reply:
"Adds" is used a lot too, just check out https://localhost:44300/Admin/Features.

Reply:
Also Provides is used a lot too. I think Provides is a better term because feature don't add but provide things.

Reply:
OK, I don't want to continue this English debate :).